test/SlowTests/Issues/RavenDB-12397.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System.Linq;
+using FastTests;
+using Newtonsoft.Json.Linq;
+using Raven.Client.Documents;
+using Xunit;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_12397 : RavenTestBase
+    {
+        public class Beer
```

Comment:
@myarichuk Try to use `private class` next time

Reply:
ok

Reply:
changed it in commit after code review